docs/development/background/getting-started-5-quick-quiz.md

```diff
@@ -0,0 +1,42 @@
+# Electron Quick Quiz
```

Comment:
"Quick Quiz"  i don't believe is the tone we want to strike here: a friendlier approach would be something like "FAQ"